lib/features/sign_up/view/sign_up_view.dart

```diff
@@ -19,9 +19,8 @@ class SignUpScreen extends StatelessWidget {
           return state is FirstLaunchScreen && state.isFirstLaunch == false;
         },
         listener: (context, state) {
-          Navigator.of(context)
-              .pushReplacement(MaterialPageRoute(builder: (context) {
-            return HomePageView();
```

Comment:
if not related to this story, please do not change

Reply:
It relates to fixing bugs of HomePageView and SignUp flow.